src/Plugin.Maui.Audio/AudioRecorder/AudioRecorder.android.cs

```diff
@@ -160,6 +144,9 @@ void WriteAudioDataToFile()
 
 		if (audioRecord is not null && outputStream is not null)
 		{
+			var header = GetWaveFileHeader(0, 0, sampleRate, channels, bitDepth);
+			outputStream.Write(header, 0, 44);
```

Comment:
I see this number `44` is referred more than once. Do you think we could place this under a constant with a suitable name? `HeaderLength` perhaps?